File: ios/Classes/core/Channels.swift
import Foundation

public struct Channels {
    private init() {}

    public static let methods = "adt_gptom/methods"
    public static let events = "adt_gptom/events"

    /// Plugin's own log lines, mirrored to Dart for on-device diagnostics.
    public static let logs = "adt_gptom/logs"
}

File: android/src/main/kotlin/cz/appsdevteam/gptom/core/Channels.kt
package cz.appsdevteam.gptom.core

object Channels {
    const val METHODS = "adt_gptom/methods"
    const val EVENTS = "adt_gptom/events"

    /** Plugin's own log lines, mirrored to Dart for on-device diagnostics. */
    const val LOGS = "adt_gptom/logs"
}
